src/commands/bundle/create.ts

```diff
@@ -0,0 +1,50 @@
+import path from "path";
+import { Command } from "@oclif/core";
+import tar from "tar";
+import { generateManifest } from "../../manifest";
+
+export default class BundleCreate extends Command {
+  static summary = "Bundles an application for use on the Armada Network.";
+  static examples = ["<%= config.bin %> <%= command.id %> my-site-v1.0.0 ./dist"];
+  static usage = "<%= command.id %> NAME DIR";
+  static aliases = ["bundle:create", "bundle"];
```

Comment:
Having both a topic and a command used to work fine in my tests, but now it's not showing up as I'd expect it. It could be because I also added aliases. Not sure if it's worth an investigation. Should we just remove the top level commands and have one use "armada bundle create" and "armada project publish"?

Reply:
Removed